docs/source/getting-started/installation.rst

```diff
@@ -27,6 +27,8 @@ To install the SDK on Mac or Linux:
      curl -sSL https://get.daml.com/ | sh
 2. If prompted, add ``~/.daml/bin`` to your PATH.
 
+   If you don't know how to do this, try following `these instructions for MacOS <https://hathaway.cc/2008/06/how-to-edit-your-path-environment-variables-on-mac/>`_ or `these instructions for Windows <https://www.java.com/en/download/help/path.xml>`_.
```

Comment:
I wonder if we should add this to the prompt message instead of in the docs (or possibly both)?

Reply:
I think it'd be a nice addition to the prompt message. As long as the links are stable that seems reasonable - @fmota-da , what do you think?